src/module_wrap.cc

```diff
@@ -486,15 +488,18 @@ void ModuleWrap::Resolve(const FunctionCallbackInfo<Value>& args) {
   }
   Utf8Value url_utf(env->isolate(), args[1]);
   URL url(*url_utf, url_utf.length());
-  // printf("resolving, args.Length() %d %s\n", args.Length(), *url_utf);
+
   if (url.flags() & URL_FLAGS_FAILED) {
     env->ThrowError("second argument is not a URL string");
     return;
   }
 
   URL result = node::loader::Resolve(*specifier_utf, &url, true);
   if (result.flags() & URL_FLAGS_FAILED) {
-    env->ThrowError("module not found");
+    std::string msg = "module ";
+    msg += *specifier_utf;
+    msg += " not found";
+    env->ThrowError(msg.c_str());
```

Comment:
Is this error smth the user will see? If so it's not consistent in style or content with the similar cjs one.